example/recursive-containers.sw

```diff
@@ -712,10 +734,9 @@ end
 def bench_insert = \i.
   // Use the given function to construct (Flat)Set from the head of provided list of lists and return the tail
   let set_from_first_list : (List a -> s) -> List (List a) -> List (List a) =\from_list.\lls.
-    case lls (\_. lls) (\ls_nlls.
-      let ns = from_list (fst ls_nlls) in
-      snd ls_nlls
-    )
+    case lls
+      (\_. lls)
+      (λmatch \_. \nlls. nlls)
```

Comment:
I just noticed that the `from_list` call got deleted here.

It's not used, because it only measures the time to go through the list of lists and create a set from each.

Reply:
Hah, and I was so proud that I had noticed and gotten rid of this stray unused code... :laughing: